plugins/camera/camera.h

```diff
@@ -0,0 +1,294 @@
+#pragma once
+
+#include <functional>
+#include <memory>
+#include <string>
+#include <vector>
+#include <map>
+#include "plugin_base.h"
+
+namespace dronecore {
+
+class CameraImpl;
+class Device;
+
+/**
+ * @brief The Camera class allows to control generic mavlink cameras.
```

Comment:
Perhaps to
> The Camera class can be used to manage cameras that implement the MAVLink Camera Protocol: https://mavlink.io/en/protocol/camera.html

Reply:
This plugin is different in that multiple instances can be used with one device. We should capture that here too.